tools/depends/target/libffi/Makefile

```diff
@@ -9,11 +9,6 @@ ARCHIVE=$(SOURCE).tar.gz
 
 # configuration settings
 CONFIGURE= ./configure --prefix=$(PREFIX) --disable-shared --disable-builddir 
-ifeq ($(OS), ios)
```

Comment:
I had two options
- disable assembler for android
- fix the assembler code, which also fixes it for ios

I'll split the libffi changes into a separate PR.